src/utils/robot_model.cpp

```diff
@@ -135,5 +135,104 @@ namespace utils {
         return pinocchio::getJointKinematicHessian(_model, _data, id, reference_frame);
     }
 
+    void RobotModel::compute_rnea_double_support(const std::unordered_map<std::string, Eigen::MatrixXd>& sensor_data,
+        const Eigen::VectorXd& q, 
+        const Eigen::VectorXd& v,
+        const Eigen::VectorXd& a, 
+        Eigen::VectorXd& tau_model,
```

Comment:
is this the result of the function? If so, we should either have it at the beginning or at the end. But I would prefer to simply return it (can be less efficient, but much clearer).

Are tau torques here?